src/components/inputs/inputs.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { colorsV3 } from '@hedviginsurance/brand'
+import { text, withKnobs } from '@storybook/addon-knobs'
+import React from 'react'
+import { withFormik } from 'storybook-formik'
+import { InputField } from './'
+
+export default {
+  title: 'Components/InputField',
+  component: InputField,
+  decorators: [withFormik, withKnobs],
+  parameters: {
+    formik: {
+      initialValues: {
+        foo: 'Some value',
+      },
+    },
+  },
+}
+
+export const Light = () => (
+  <div style={{ maxWidth: '320px', margin: '40px auto' }}>
+    <InputField
+      label={text('Label', '')}
+      name="foo"
+      type="text"
+      autoComplete="off"
+      placeholder=""
+      touched={false}
+      errors={text('Error text', '')}
+    />
+  </div>
+)
+
+export const Dark = () => (
+  <div style={{ maxWidth: '320px', margin: '40px auto' }}>
+    <InputField
+      label={text('Label', '')}
+      name="foo"
+      type="text"
+      autoComplete="off"
+      placeholder=""
+      touched={false}
+      errors={text('Error text', '')}
+      variant="dark"
+    />
+  </div>
+)
```

Comment:
Maybe create a generic one that takes the variant as input and then return that for Light/Dark

Reply:
Although I can see now these are just test examples, I guess this is better then.